resources.yaml

```diff
@@ -634,18 +662,41 @@ items:
                     periodSeconds: 10
                     successThreshold: 1
                     timeoutSeconds: 1
+                    initialDelaySeconds: 20
+                - name: oc-proxy
+                  image: docker.io/radanalyticsio/oc-proxy:stable
```

Comment:
docker.io should be dropped here to match the other images.
Having docker.io there seems to change the console display (no tag) in origin 1.5.1 (and maybe other versions)